gulpfile.js

```diff
@@ -147,14 +154,15 @@ gulp.task('cachebust', cb => {
 		.on('end', () => {
 			delete require.cache[require.resolve(REV_PATH)];
 			manifest = require(REV_PATH);
-
 			swagList.forEach(swag => {
-				const filename = `swag-img/${swag.image.split('/').pop()}`;
-				if (!manifest[filename]) {
-					console.warn('Unable to find image in manifest:', filename);
-					return;
-				}
-				swag.image = `/assets/${manifest[filename]}`;
+				Object.entries(swag.image).forEach(([extension, filename]) => {
```

Comment:
Wouldn't you be trying to destructure a string here?

Reply:
Nope, see here: https://github.com/swapagarwal/swag-for-dev/pull/226/commits/1ae989bf18b8e03b9fa3fba59c28051cfa93dd29#diff-b9e12334e9eafd8341a6107dd98510c9R118. Just switched to `images` to be clearer.

Reply:
whoops, I thought you were using `Object.values` - I think that would be better here?

Reply:
I reuse the extension from the object keys. If we want to use Object values I'll have to compute it from filename like so: 

```diff
diff --git a/gulpfile.js b/gulpfile.js
index af6b475..668cf4c 100644
--- a/gulpfile.js
+++ b/gulpfile.js
@@ -155,8 +155,9 @@ gulp.task('cachebust', cb => {
                        delete require.cache[require.resolve(REV_PATH)];
                        manifest = require(REV_PATH);
                        swagList.forEach(swag => {
-                               Object.entries(swag.images).forEach(([extension, filename]) => {
+                               Object.values(swag.images).forEach(filename => {
                                        filename = `swag-img/${filename.split('/').pop()}`;
+                                       const extension = filename.split('.').pop();
                                        if (!manifest[filename]) {
                                                console.warn('Unable to find image in manifest:', filename);
                                                return;
```

Reply:
Gonna keep `Object.entries` as it's fits best